lumicks/pylake/detail/image.py

```diff
@@ -3,6 +3,60 @@
 import numpy as np
 
 
+class ImageMetaData:
```

Comment:
Metadata is usually one word, so `ImageMetadata`.

Reply:
Amended. 👍